### DISCUSSION TITLE
Can't go back to Normal mode after completion

### DISCUSSION THREAD

**Discussion by italoaa** [minad/corfu#434] [Q&A]
Currently this is my evil, evil-collection and corfu configurations:
``` elisp
(use-package evil
  :demand t
  :init
  (setq evil-want-integration t
        evil-want-keybinding nil
        evil-want-C-u-scroll t
        evil-vsplit-window-right t
        evil-split-window-below t)

  (setq evil-undo-system 'undo-redo)
  (evil-mode))

(use-package evil-collection
  :after evil
  :config
  (evil-collection-init))

(use-package corfu
  ;; Optional customizations
  :custom
  (corfu-auto t)
  (corfu-auto-prefix 2)
  (corfu-auto-delay 0.25)
  (tab-always-indent 'complete)

  :init
  (global-corfu-mode))
```

Everything is perfect until the corfu UI pops up and I get to choose between the options. From now on the escape key seems to not be bound to anything. Before the corfu UI it is bound to (evil-force-normal-state) but once the UI appears the only keymap I see is "ESC ESC ESC" or "M-ESC ESC" it is bound to (keyboard-escape-quit)

I appreaciate all the help I could get thank you I love the packageeee and the mentality.

**Comment by minad** [comment]
Hi @italoaa! Thanks for the kind words. I cannot help you with Evil, since I don't use it. I suggest you ask on the [evil-collection](https://github.com/emacs-evil/evil-collection/discussions) repository for help, since the Evil/Corfu integration lives there.

**Reply by italoaa** [reply]
Hi @minad! So glad to hear from you. I completly understand because I think this is an evil issue, thank you anyways for the help and the direction. Keep up the AMAZING WORK :)
